src/views/CountryNsOverviewContextAndStructure/NationalSocietyIndicators/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { useOutletContext } from 'react-router-dom';
+import { isDefined, isNotDefined } from '@togglecorp/fujs';
+
+import BlockLoading from '#components/BlockLoading';
+import Container from '#components/Container';
+import TextOutput from '#components/TextOutput';
+import useTranslation from '#hooks/useTranslation';
+import { sumSafe } from '#utils/common';
+import { type CountryOutletContext } from '#utils/outletContext';
+import { useRequest } from '#utils/restRequest';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+function NationalSocietyIndicators() {
+    const strings = useTranslation(i18n);
+
+    const { countryId } = useOutletContext<CountryOutletContext>();
+
+    const {
+        pending: indicatorPending,
+        response: indicatorResponse,
+    } = useRequest({
+        url: '/api/v2/country/{id}/databank/',
+        skip: isNotDefined(countryId),
+        pathVariables: {
+            id: isDefined(countryId) ? Number(countryId) : undefined,
+        },
+    });
+
+    const staffValue = sumSafe([
+        indicatorResponse?.female_staff_total,
+        indicatorResponse?.male_staff_total,
+    ]);
+
+    const volunteersValue = sumSafe([
+        indicatorResponse?.male_volunteer_total,
+        indicatorResponse?.female_volunteer_total,
+    ]);
```

Comment:
what about `other` staff?

Reply:
we should use `KPI_PStaff_Tot, KPI_PeopleVol_Tot` indicators

Reply:
@samshara lets note that we are including 3rd gender population.